src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -1505,7 +1506,13 @@ public void operate() {
 
         } catch(VoldemortException e) {
             response.setError(ProtoUtils.encodeError(errorCodeMapper, e));
-            logger.error("handleGetMetadata failed for request(" + request.toString() + ")", e);
+            String errorMessage = "handleGetMetadata failed for request(" + request.toString() + ")";
+            
+            if(e instanceof StoreNotFoundException) {
```

Comment:
Nitpick: Two catch blocks with two lines each, only one of which is duplicated, seems cleaner to me than trying to DRY this code by using `instanceof`.